app.go

```diff
@@ -413,92 +410,178 @@ func Serve() {
 		app.cfg.Database.Database = "writefreely"
 	}
 
+	// TODO: check err
 	connectToDatabase(app)
```

Comment:
Should we check this error before merging?

Reply:
We'd actually need to refactor `connectToDatabase()` to return an error -- right now it does `os.Exit(1)` when encountering errors, instead. Definitely want to fix it, but I'd say we can save it for a little later.

Reply:
:+1: